content/uc-doc/api_sdk/rest_api/confd/user_management.md

```diff
@@ -0,0 +1,291 @@
+---
+title: User Management
+---
+
+The purpose of this page is to explain how to set up a fully operational “telephony” user (with the
+Wazo API), having the following features/properties:
+
+- Voicemail enabled
+- Forward enabled
+- Credentials for API usage
+- One or more lines (including extensions and SIP endpoints) created
+- One or more created DID numbers that allow(s) receiving calls
+- Member of a ring groups
+- …
+
+Want more information about the user's API, consult the
+[API documentation](/documentation/api/configuration.html#tag/users/operation/create_user).
+
+## Create User (the new way)
+
+As you may see in the previous chapter, there are a lot of steps to follow, in a specific order, to
+be able to correctly set up a new “telephony” user with all main features enabled.
+
+To help developers to quickly create users and easily understand the information needed, the user
+creation process has been revamped. This allows developers to use one single endpoint, instead of
+having to call many endpoints. Also relations and complete validation/rollback are handled at the
+server-side:
+
+- the attachments (the link between the user and their related resources)
+- the rollback in case of errors (if one of the resource creations fails)
+
+**Warning**: The following resources must be created before, when needed:
+
+- Switchboards
+- Groups
+- Functions Keys Templates
```

Comment:
The API supports creating those resources on the fly no ? 
We don't really need them to be created before the POST on `/users` from what i understood

Reply:
I confirm: these resources must be created before if we want them to be linked to the user. In this case, when we create a user, we provide their IDs only.